src/importexport/guitarpro/internal/importgtp.cpp

```diff
@@ -2940,9 +2941,9 @@ Score::FileError importGTP(MasterScore* score, const QString& name)
     // otherwise it's an older version - check the header
     else if (strcmp(&header[1], "FIC") == 0) {
         uchar l;
-        fp.read((char*)&l, 1);
+        fp.read((uint8_t*)&l, 1);
         char ss[30];
-        fp.read(ss, 30);
+        fp.read(reinterpret_cast<uint8_t*>(ss), 30);
```

Comment:
Mind to explay why using different ways of casting here?

Reply:
I did same )